__tests__/linkdrop.spec.ts

```diff
@@ -0,0 +1,125 @@
+import { Runner, toYocto, createKeyPair, BN, tGas } from "..";
+
+describe(`Running on ${Runner.getNetworkFromEnv()}`, () => {
+  jest.setTimeout(60000);
+  let runner: Runner;
+
+  beforeAll(async () => {
+    runner = await Runner.create(async ({ runtime }) => ({
+      linkdrop: await runtime.createAndDeploy(
+        "linkdrop",
+        `${__dirname}/build/debug/linkdrop.wasm`
+      ),
+    }));
+  });
+
+  test("call myself", async () => {
+    await runner.run(async ({ root, linkdrop }) => {
+      // Create temporary keys for access key on linkdrop
+      const senderKey = createKeyPair();
+      const public_key = senderKey.getPublicKey().toString();
+
+      // This adds the key as a function access key on `create_account_and_claim`
+      await root.call(
```

Comment:
Can `linkdrop` call itself here? That would seem more accurate to me. `linkdrop.call(linkdrop, 'send', …)`

Reply:
No because you are attaching a deposit.  And ideally this is how the contract works.